css/src/components/form/input.scss

```diff
@@ -5,7 +5,7 @@ $input-border-bottom-color: $control-border-bottom !default;
 $input-placeholder-color: $text-subtle !default;
 
 $input-hover-color: $text !default;
-$input-hover-border-color: $secondary !default;
+$input-hover-border-color: $default-hover !default;
 $input-focus-border-color: $primary !default;
```

Comment:
This comment is not related to current file, posting to have a thread.

Is it possible to update button's border color to `control-border`? It will look better in terms of transition from default to hover state and will match fluent better. It's quite confusing why `text-subtle` was used on it in a first place so maybe it's a good time to fix that

![image](https://github.com/user-attachments/assets/fbdc97c5-c381-48d9-95b9-e2146067754a)

Reply:
Updated